src/components/capture/creation/DetailsForm.tsx

```diff
@@ -22,12 +22,9 @@ function NewCaptureDetails(props: NewCaptureDetailsProps) {
     const intl = useIntl();
     const { state, dispatch, readonly, displayValidation } = props;
 
-    const {
-        data: { connectors },
-        loading,
-        error,
-    } = useConnectors();
+    const { data: connectorsData, isError, isSuccess, error } = useConnectors();
```

Comment:
This is how the new hooks are consumed in forms... just using the `useAsync` response.